File: Multi_Language_CodeExamples/java/Person.java
public class Person {
    private String name;
    private int age;
    
    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }
    
    public void greet() {
        System.out.println("Hello, I'm " + name + " and I'm " + age + " years old");
    }
    
    public static void main(String[] args) {
        Person person = new Person("Alice", 25);
        person.greet();
    }
}

File: Multi_Language_CodeExamples/csharp/Person.cs
using System;

class Person {
    public string Name { get; set; }
    public int Age { get; set; }
    
    public void Greet() {
        Console.WriteLine($"Hello, I'm {Name} and I'm {Age} years old");
    }
    
    static void Main() {
        Person person = new Person { Name = "Bob", Age = 30 };
        person.Greet();
    }
}
